synapse/storage/databases/main/user_directory.py

```diff
@@ -497,8 +497,7 @@ async def add_users_who_share_private_room(
     async def add_users_in_public_rooms(
         self, room_id: str, user_ids: Iterable[str]
     ) -> None:
-        """Insert entries into the users_who_share_private_rooms table. The first
-        user should be a local user.
+        """Insert entries into the users_in_public_rooms table.
```

Comment:
Was this comment just stale?

Reply:
I think it was a copy/paste error from the method above (`add_users_who_share_private_room`).